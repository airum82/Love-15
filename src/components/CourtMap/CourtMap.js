import { GoogleApiWrapper, Map, Marker } from 'google-maps-react';
import { mapsKey } from '../../APIkey';
import React from 'react';

export const CourtMap = ({coord, google}) => {
  return (
    <Map
      google={google}
      zoom={14}
      initialCenter={coord}
      center={coord}
      style={{
        width: '40vw',
        height: '40vh',
      }}
    >
      <Marker
        position={coord}
      />
    </Map>
  )
}

export default GoogleApiWrapper({
  apiKey: mapsKey
})(CourtMap);