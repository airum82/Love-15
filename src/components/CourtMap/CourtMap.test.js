import { shallow } from 'enzyme';
import { CourtMap } from './CourtMap';
import React from 'react';

describe('CourtMap', () => {

  it('should match snapshot upon render', () => {
    const coord = {
      lat: 5.36,
      lng: 5.47
    }
    const google = {};
    const wrapper = shallow(
      <CourtMap
       coord={coord}
       google={google}
      />
    )
    expect(wrapper).toMatchSnapshot();
  })
})