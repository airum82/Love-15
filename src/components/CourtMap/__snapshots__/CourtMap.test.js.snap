// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CourtMap should match snapshot upon render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    coord={
        Object {
            "lat": 5.36,
            "lng": 5.47,
          }
    }
    google={Object {}}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "center": Object {
        "lat": 5.36,
        "lng": 5.47,
      },
      "centerAroundCurrentLocation": false,
      "children": <Marker
        name="Marker"
        position={
                Object {
                        "lat": 5.36,
                        "lng": 5.47,
                      }
        }
/>,
      "containerStyle": Object {},
      "google": Object {},
      "initialCenter": Object {
        "lat": 5.36,
        "lng": 5.47,
      },
      "style": Object {
        "height": "50vh",
        "margin": "3% auto",
        "width": "60vw",
      },
      "visible": true,
      "zoom": 14,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "name": "Marker",
        "position": Object {
          "lat": 5.36,
          "lng": 5.47,
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "center": Object {
          "lat": 5.36,
          "lng": 5.47,
        },
        "centerAroundCurrentLocation": false,
        "children": <Marker
          name="Marker"
          position={
                    Object {
                              "lat": 5.36,
                              "lng": 5.47,
                            }
          }
/>,
        "containerStyle": Object {},
        "google": Object {},
        "initialCenter": Object {
          "lat": 5.36,
          "lng": 5.47,
        },
        "style": Object {
          "height": "50vh",
          "margin": "3% auto",
          "width": "60vw",
        },
        "visible": true,
        "zoom": 14,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "name": "Marker",
          "position": Object {
            "lat": 5.36,
            "lng": 5.47,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
